src/components/utils.js

```diff
@@ -27,6 +27,26 @@ export const validatePassword = function(password) {
     return password.length === 0;
 };
 
+    /**
+     * Return true if user input current password correctly
+     */
+export const validateCurrentPassword = function(password) {
```

Comment:
Maybe we should put this method in `api.js` since it sends a request to the back-end and requires access to the client.